src/components/EntityInfo/EntityHeader.vue

```diff
@@ -129,22 +131,14 @@ export default class EntityHeader extends Vue {
     }
   }
 
-  /** The incorporation/registration/continuationIn/amalgamation application description. */
-  get appDescription (): string {
-    if (this.isAmalgamationTodo) {
-      return this.getTasks[0]?.task.filing.displayName
-    }
-    if (this.isAmalgamationFiling) {
-      return this.getFilings[0]?.displayName
-    }
+  /** The amalgamation / continuation in / incorporation application / registration description. */
+  get bootstrapFilingDescription (): string {
+    if (this.isAmalgamationTodo) return this.getTasks[0]?.task.filing.displayName
+    if (this.isAmalgamationFiling) return this.getFilings[0]?.displayName
 
-    if (this.isContinuationInTodo) {
-      return this.getTasks[0]?.task.filing.displayName
-    }
-    // *** TODO: what needs to be done here for pending continuation bootstrap filing?
-    if (this.isContinuationInFiling) {
-      return this.getFilings[0]?.displayName
-    }
+    if (this.isContinuationInTodo) return this.getTasks[0]?.task.filing.displayName
+    if (this.isContinuationInPending) return this.getPendingsList[0].displayName
+    if (this.isContinuationInFiling) return this.getFilings[0]?.displayName
```

Comment:
^^ get Display Name from correct list

Example when filing is in Pending list:

![image](https://github.com/user-attachments/assets/f980b8da-c80c-4f4c-a76b-ff5e040ef40c)